spec/plans/execute.fmf

```diff
@@ -318,3 +291,54 @@ description: |
                     systemctl start httpd
                     echo foo > /var/www/html/index.html
                     curl http://localhost/ | grep foo
+
+/sync:
+    summary: Synchronization between steps or tests
+    description: |
+        The main idea of :ref:`/spec/plans/provision/multihost`
+        tests is to run certain tasks in parallel (e.g., a RestAPI
+        requests from a client to a server). Therefore, these kinds
+        of tests may require test synchronization.
+        The synchronization may be specified for specific steps
+        by providing a list of step names under ``step`` keyword.
+        This creates a synchronization
+        barrier before each provided step.
+
+        There is also an option for a synchronization on the test
+        level defined using ``test`` keyword accepting several values:
+
+            each
+                Synchronize execution of all tests.
+            names
+                A list of test names executed on both hosts that
+                should be synchronized before running.
+            indexes
+                A list of indexes into the list of tests, which labels
+                specific tests for synchronization.
+            none
+                **Default** value, which disables the synchronization
```

Comment:
Default value `none` seems to me too kernel specific and not intuitive in general. For us the `each` would considered more as a default. I suggest to not have a default value and have the `test` key mandatory.

Reply:
It depends on a POV. I would vote for something like `first` or `start` which would sync a start of a "recipe", the resp would be on the tests as we do in beaker.

Reply:
The currently proposed keywords are here https://github.com/teemtee/tmt/issues/1705#issuecomment-1326348409. Let's move the conversation here.